src/Controller/Api/Cliente/AtivarOuDesativarClienteController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Controller\Api\Cliente;
+
+use App\Repository\ClienteRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/cliente/ativar', name: 'toggle')]
+class AtivarOuDesativarClienteController extends AbstractController
+{
+    private ClienteRepository $clienteRepository;
+
+    public function __construct(ClienteRepository $clienteRepository)
+    {
+        $this->clienteRepository = $clienteRepository;
+    }
+
+    
+    #[Route('/verificar/ativar/{id}', name: '_verificar_ativar')]
+    public function __invoke(int $id): RedirectResponse
+    {
+        $cliente = $this->clienteRepository->find($id);
+
+        if (!$cliente) {
+            throw $this->createNotFoundException('Cliente não encontrado.');
```

Comment:
```suggestion
            throw $this->clienteNotFoundException('Cliente não encontrado.');
```
não era pra ser Cliente não? se sim lembra de trocar o resto